addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1677,6 +1678,9 @@ export class OdooEditor extends EventTarget {
             // Do not apply commands out of the editable area.
             return false;
         }
+        if (!isSelectionInContentEditable(sel)) {
+            return false;
+        }
```

Comment:
Actually, if you change the condition above your new code to use `isSelectionInEditable` instead, I think it would cover all cases.

Reply:
Indeed, using `isSelectionInEditable` solved all the problems